data/events/data/2021-09-22.js

```diff
@@ -0,0 +1,20 @@
+module.exports = () => {
+  return {
+    id: "2021-09-22",
+    title: "LeedJS is back....",
+    blurb: `We are back with a hybrid offfering and some great talks. Please do come along or tune in online and get back in touch with the community.`,
+    talks: [
+      "augmented-reality-with-javascript",
+      "how-kids-are-using-the-internet-in-2021",
+    ],
+    sponsors: ["bjss"],
+    prizes: [],
+    date: "2021-09-22",
+    start_time: "17:30",
+    end_time: "19:00",
+    ticket_date: "2021-10-10",
+    ticket_link: "https://share.hsforms.com/1Tta4n_yWSgSDfVKqbnVI3w5lgth",
+    announce_date: "2021-09-10",
```

Comment:
So the ticket and announce dates are used by the automation to send out emails. You might want to set the announce date to today/tomorrow so that emails get sent. You might also need to modify the template for the emails in website/site/automation/

Reply:
Thanks @CodeFoodPixels  - will do this!

Reply:
I have resolved these issues, but somehow Netflify builds are failing for some unknown reason?